<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallPopUp</class>
 <widget class="QMainWindow" name="CallPopUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string>Compose your message...</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(28, 28, 28);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QListWidget::item:selected {
	background: none;
	color: none;
}

QListWidget::item:hover {
	background: none;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="6">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gadugi</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: silver;</string>
      </property>
      <property name="text">
       <string>ContactName</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSlider" name="VolumeSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="EndcallButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>61</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>61</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gadugi</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Call your contact</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: silver;
border-width: 2px;
border-style: solid;
border-color: rgb(52, 52, 52);
padding: 4px;
}

QPushButton:pressed{
padding: 2px;
}
</string>
      </property>
      <property name="text">
       <string>End Call</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="6" colspan="6">
     <widget class="QWidget" name="VideoWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>350</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gadugi</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:silver;</string>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuInput">
    <property name="font">
     <font>
      <family>Gadugi</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:silver;</string>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOutPut">
    <property name="title">
     <string>OutPut</string>
    </property>
   </widget>
   <addaction name="menuInput"/>
   <addaction name="menuOutPut"/>
  </widget>
  <action name="actionTheme">
   <property name="text">
    <string>Theme</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionSelect_color">
   <property name="text">
    <string>Select color</string>
   </property>
  </action>
  <action name="actionStyle">
   <property name="text">
    <string>Style</string>
   </property>
   <property name="font">
    <font>
     <family>Gadugi</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionDisconnect_2">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionOnline">
   <property name="text">
    <string>Online</string>
   </property>
  </action>
  <action name="actionDo_Not_Disturb">
   <property name="text">
    <string>Do Not Disturb</string>
   </property>
  </action>
  <action name="actionOffline">
   <property name="text">
    <string>Offline</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
